### ISSUE TITLE
`fly machine restart` not working with `--time` argument

### ISSUE THREAD

**Issue by wjordan** [superfly/flyctl#3342]
```
$ fly machine restart --help | grep time
      --time int             Seconds to wait before killing the machine
```

```
$ fly machine restart --time 10
Restarting machine [id]
Error: failed to restart machine [id]: could not stop machine [id]: failed to restart VM [id]: invalid_argument: invalid RestartMachineRequest.Timeout: value must be inside range [1s, 1m0s] (Request ID: [req])
$ fly machine restart --time 59
Restarting machine [id]
Error: failed to restart machine [id]: could not stop machine [id]: failed to restart VM [id]: timeout must not exceed 60 seconds (Request ID: [req])
```

**Comment by wjordan** [comment]
```
$ LOG_LEVEL=DEBUG fly machine restart --time 19

[...]
Restarting machine [id]
DEBUG --> POST https://api.machines.dev/v1/apps/fly-metrics-staging/machines/[id]/restart?force_stop=false&timeout=19000000000

DEBUG <-- 400 https://api.machines.dev/v1/apps/fly-metrics-staging/machines/[id]/restart?force_stop=false&timeout=19000000000 (15.05ms)

DEBUG {
  "error": "timeout must not exceed 60 seconds"
}

```

**Comment by wjordan** [comment]
Traced this back to https://github.com/superfly/flyctl/pull/1318 - it looks like `RestartMachineInput.Timeout` is a `time.Duration` but is passed along directly as an 'integer seconds' query param- so it expects its Duration input as an integer-nanosecond duration instead of a full-second duration.

This was how this command was initially released in #1338, but looks like it was changed in [#2039](https://github.com/superfly/flyctl/pull/2039/files#diff-b41aa36dc39174361c13fa3a5f6f7076e99e67a7c5e8e22cc90d79cdfda6a65fL70-R70) and probably hasn't been working since then.
